src/alire/alire-properties-licenses.ads

```diff
@@ -1,11 +1,17 @@
 with Alire.Licensing;
+with Alire.TOML_Keys;
 
 package Alire.Properties.Licenses is
 
    function Image (L : Licensing.Licenses) return String;
 
+   pragma Warnings (Off);
+   function Key (L : Licensing.Licenses) return String is (TOML_Keys.License);
+   pragma Warnings (On); -- Unreferenced => L
```

Comment:
Protip: rename `L` to `Dummy` (or `Dummy_L`), and the warning will go away.

Reply:
Is this some gnat magic? :D

Reply:
Yes indeed. ;-) See https://docs.adacore.com/gnat_rm-docs/html/gnat_rm/gnat_rm/implementation_defined_pragmas.html#pragma-unmodified